presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestingOracleServer.java

```diff
@@ -36,17 +36,28 @@ public TestingOracleServer()
     {
         super("wnameless/oracle-xe-11g-r2");
 
-        // this is added to allow more processes on database, otherwise the tests end up giving a ORA-12519
-        // to fix this we have to change the number of processes of SPFILE
-        // but this command needs a database restart and if we restart the docker the configuration is lost.
-        // configuration added:
-        // ALTER SYSTEM SET processes=500 SCOPE=SPFILE
-        // ALTER SYSTEM SET disk_asynch_io = FALSE SCOPE = SPFILE
-        this.addFileSystemBind("src/test/resources/spfileXE.ora",
-                "/u01/app/oracle/product/11.2.0/xe/dbs/spfileXE.ora",
-                BindMode.READ_ONLY);
-
         start();
+
+        try (Connection connection = DriverManager.getConnection(getJdbcUrl(), super.getUsername(), super.getPassword());
```

Comment:
```suggestion
        try (Connection connection = DriverManager.getConnection(getJdbcUrl(), getUsername(), getPassword());
```

-- you didn't need `super`, did you?